gateway/managers/syncman/syncman_crud.go

```diff
@@ -542,12 +542,8 @@ func (s *Manager) GetCollectionRules(ctx context.Context, project, dbAlias, col
 	return http.StatusOK, []interface{}{coll}, nil
 }
 
-type dbJSONSchemaResponse struct {
-	Fields []*model.FieldType `json:"fields"`
-}
-
 // GetSchemas gets schemas from config
-func (s *Manager) GetSchemas(ctx context.Context, project, dbAlias, col, format string, params model.RequestParams) (int, []interface{}, error) {
+func (s *Manager) GetSchemas(ctx context.Context, project, dbAlias, col, format string, params model.RequestParams) (int, []DbSchemaResponse, error) {
```

Comment:
Keep it array of interface or it will cause a problem with the integration module
```suggestion
func (s *Manager) GetSchemas(ctx context.Context, project, dbAlias, col, format string, params model.RequestParams) (int, []interface{}, error) {
```